### PR TITLE
Add osx build

### PR DESCRIPTION
**Pull request by sodre**
As discussed in the review of conda-forge/staged-recipes#5409, investigate building the OSX kubernetes client libraries.

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: True  # [not linux]
-  number: 0
+  skip: True  # [win]
+  number: 1
```
Comment (sodre) [inline recipe/meta.yaml]: Build number has to be bumped because kubefed was not originally included int the linux build.

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
FYI can just do `which` for commands that are non-trivial to run.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -2,36 +2,66 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
+  sha256: {{ sha256 }}  # [not win]
+
+  fn: {{ name }}-client-{{ version }}.tar.gz  # [win]
+  sha256: {{ sha256_win64 }}  # [win]
 
 build:
-  skip: True  # [not linux]
-  number: 0
+  skip: True  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Might want to skip Windows 32-bit whenever Windows is enabled if these are 64-bit packages only.

Comment (sodre) [reply recipe/meta.yaml]: I will take care of it in the windows PR.

**Pull request merged**